src/app/shared/font-picker/font-picker.component.scss

```diff
@@ -0,0 +1,4 @@
+#font-selector {
+    max-width: 4rem;
+   }
```

Comment:
There shouldn't be so much space between the font selector and notifications icon. Please correct.